packages/about-you/composables/__tests__/composables/useProduct/index.spec.ts

```diff
@@ -36,4 +40,59 @@ describe.skip('[about-you-composables] useProduct', () => {
       term: searchParams.term
     });
   });
+
+  it('map available filters  ', async () => {
+    (getFilters as jest.Mock).mockReturnValueOnce([
+      {
+        id: null,
+        slug: 'prices',
+        name: 'Prices',
+        type: 'range',
+        values: [
+          {
+            min: 5000,
+            max: 10900,
+            productCount: 4
+          }
+        ]
+      },
+      {
+        id: 1525,
+        slug: 'country',
+        name: 'Country',
+        type: 'attributes',
+        values: [
+          {
+            name: 'Germany',
+            productCount: 1,
+            id: 73349
+          }
+        ]
+      }
+    ]);
+    const mappedFilters = await factoryParams.availableFilters({categoryId: 1111});
+
+    expect(getFilters).toHaveBeenCalled();
+    expect(mappedFilters).toEqual({
```

Comment:
when checking objects we often want to use `expect(...).objectContaining` with the most important parameters instead. Doesn't necessarily apply here, just a side note